### PR TITLE
owner: modify resource manager owner to PD 

### PR DESCRIPTION
**Pull request by HuSharp**
Signed-off-by: husharp <jinhao.hu@pingcap.com>

modify resource manager owner to PD

### CODE REVIEW

**Comment by HuSharp** [comment]
@nolouch @rleungx @JmPotato  PTAL, thx!

**Review by rleungx — APPROVED** [review]

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -20,7 +20,7 @@ aliases:
   sig-approvers-pb-coprocessor: [cfzjywxk]
   sig-approvers-pb-deadlock: [MyonKeminta, cfzjywxk]
   sig-approvers-pb-debug: [cfzjywxk]
-  sig-approvers-pb-pd: [niubell, rleungx]
+  sig-approvers-pb-pd: [niubell, rleungx, nolouch]
   sig-approvers-pb-raftstore: [zhangjinpeng1987, overvenus]
```
Comment (wuhuizuo) [inline OWNERS_ALIASES:24]:
```suggestion
  sig-approvers-pb-raftstore: [zhangjinpeng87, overvenus]
```

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -20,8 +20,8 @@ aliases:
   sig-approvers-pb-coprocessor: [cfzjywxk]
   sig-approvers-pb-deadlock: [MyonKeminta, cfzjywxk]
   sig-approvers-pb-debug: [cfzjywxk]
-  sig-approvers-pb-pd: [niubell, rleungx]
-  sig-approvers-pb-raftstore: [zhangjinpeng1987, overvenus]
+  sig-approvers-pb-pd: [niubell, rleungx, nolouch]
+  sig-approvers-pb-raftstore: [zhangjinpeng87, overvenus]
   sig-approvers-pb-tikv: [zhangjinpeng1987]
```
Comment (wuhuizuo) [inline OWNERS_ALIASES:25]:
```suggestion
  sig-approvers-pb-tikv: [zhangjinpeng87]
```

**Review by JmPotato — APPROVED** [review]

**Review by overvenus — APPROVED** [review]

**Review by zhangjinpeng87 — APPROVED** [review]
LGTM

**Pull request merged**